client/app/style.css

```diff
@@ -8,6 +8,8 @@
   font-family: 'Inter', system-ui, sans-serif;
   color-scheme: light dark;
   font-size: 14px;
+
+  @apply dark:text-white;
```

Comment:
There's already an `@apply ...` line in this selector. You can add this at the end; ie,

```css
:root {
  @apply bg-zinc-50 dark:bg-neutral-900 dark:text-white;
  ...
}
```